common/src/main/java/fi/hel/verkkokauppa/common/rest/dto/configuration/MerchantDto.java

```diff
@@ -0,0 +1,15 @@
+package fi.hel.verkkokauppa.common.rest.dto.configuration;
```

Comment:
I added these DTO:s related to the configurations to common package, because CommonServiceConfigurationClient is used across all the core microservices. It is much clearer to have defined DTO:s so that there is a contract of what data the CommonServiceConfigurationClient is returning. For now I have only added this to MerchantDto, because it was a new method I added. I suggest we refactor at somepoint the NamespaceModel to have the defined DTO:s here in common package also.